src/storage/fs/FileSystem.ts

```diff
@@ -0,0 +1,7 @@
+export interface FileSystem {
+  readonly basePath: string
+
+  exists(path: string): Promise<boolean>
+  write(path: string, data: string): Promise<void>
+  read(path: string): Promise<string>
```

Comment:
I wanted to suggest preserving `fs` API and not creating our own, which would be better, but I see there are tiny differences between them.

Reply:
Yeah there's some differences between the NodeJS and RN API so I created a very simple interface that works for both systems. NodeJS does not have `exists` which I think is nice to have